services/cirrus/cirrus.service.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const Joi = require('joi')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const { BaseSvgScrapingService } = require('..')
+
+const schema = Joi.object({
+  message: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('unknown'))
+    .required(),
+}).required()
+
+module.exports = class Cirrus extends BaseSvgScrapingService {
```

Comment:
It was mentioned over on https://github.com/cirruslabs/cirrus-ci-docs#283 that there's a JSON endpoint available for this data. It would probably be better/simpler to use that instead of the SVG scraping

Reply:
Right, just borrowed some code from travis 😅

Reply:
No worries! And I know this is still a work in progress too. Happy to help if you have any questions along the way